Base/usr/share/man/man1/modload.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+modload - loads a kernel module
```

Comment:
This line should be indefinite/imperative, like in commit messages: "load a kernel module"